src/constants/ingestion/constants.ts

```diff
@@ -32,10 +32,20 @@ export const INGESTION_VALIDATIONS = {
     pattern: '^[0-9]$|^[1-9][0-9]$|^(100)$',
     description: 'Classification value between 0 and 100',
   },
-  fileNames: {
+  gpkgfileName: {
     pattern: '^.+\\.[Gg][Pp][Kk][Gg]$',
     description: 'File name must end with .gpkg',
   },
+  metadataShapefileFilePath: {
+    // eslint-disable-next-line no-useless-escape
+    pattern: '^((\\/[\\w-]+)*)\\/ShapeMetadata\\.shp$',
+    description: 'Valid metadata shapefile file path',
+  },
+  productShapefileFilePath: {
+    // eslint-disable-next-line no-useless-escape
+    pattern: '^((\\/[\\w-]+)*)\\/Product\\.shp$',
+    description: 'Valid product shapefile file path',
+  },
```

Comment:
lets handle error messages inside `INGESTION_VALIDATIONS`. we should separate the error message from the description